lib/Model.js

```diff
@@ -44,31 +44,37 @@ class Model {
       return;
     }
 
-    advisory.findings.forEach((finding) =>
-      finding.paths.forEach((path) => {
-        if (this.allowlist.paths.includes(`${advisory.id}|${path}`)) {
-          this.allowlistedPathsFound.push(`${advisory.id}|${path}`);
-        }
-      })
-    );
+    if (advisory.findings) {
+      advisory.findings.forEach((finding) =>
+        finding.paths.forEach((path) => {
+          if (this.allowlist.paths.includes(`${advisory.id}|${path}`)) {
+            this.allowlistedPathsFound.push(`${advisory.id}|${path}`);
+          }
+        })
+      );
 
-    if (
-      advisory.findings.every((finding) =>
-        finding.paths.every((path) =>
-          this.allowlist.paths.includes(`${advisory.id}|${path}`)
+      if (
+        advisory.findings.every((finding) =>
+          finding.paths.every((path) =>
+            this.allowlist.paths.includes(`${advisory.id}|${path}`)
+          )
         )
-      )
-    ) {
-      return;
+      ) {
+        return;
+      }
     }
 
     this.advisoriesFound.push(advisory);
   }
 
   load(parsedOutput) {
-    Object.values(parsedOutput.advisories).forEach((a) => this.process(a));
+    if (parsedOutput.advisories) {
```

Comment:
Can you add a comment here to say NPM 6 (and below add a comment to say NPM 7)

Reply:
Done